FRINXio/frinx-frontend
# Fr 306 mpls topology

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FR-306` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |


HOW TO TEST:

test with https://github.com/FRINXio/frinx-inventory-server/pull/456

you should see new mpls layer in device topology
